opm/material/fluidmatrixinteractions/EclEpsScalingPoints.hpp

```diff
@@ -199,10 +199,10 @@ struct EclEpsScalingPointsInfo
             Swu = 0.0;
             Swcr = 0.0;
             if (!sgofTables.empty())
-                extractUnscaledSgof_(sgofTables[satRegionIdx]);
+                extractUnscaledSgof_(sgofTables.getTable<SgofTable>(satRegionIdx));
```

Comment:
hm, this seems to be a slightly weird API to me: the type of the table is already implicitly specified by the respective `get*()` method of the `TableManager`, yet the class type needs to be explicitly specified here again. wouldn't it be better to do this directly in the `TableManager`? I mean an API similar to this:

```c++
tableManager.get<SgofTable>(satRegionIdx);
```